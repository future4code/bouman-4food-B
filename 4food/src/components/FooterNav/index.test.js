import React from 'react';
import renderer from 'react-test-renderer';
import { FooterNav } from './index'

it('Snapshot FooterNav', ()=> {
    const tree = renderer
    .create(<FooterNav></FooterNav>)
    .toJSON();
    expect(tree).toMatchSnapshot()
})