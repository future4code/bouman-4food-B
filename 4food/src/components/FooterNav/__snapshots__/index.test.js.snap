// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot FooterNav 1`] = `
<div
  className="sc-bdVaJa gpzSpl"
>
  <buttom>
    <img
      src="homepage.svg"
    />
  </buttom>
  <buttom>
    <img
      src="shopping-cart.svg"
    />
  </buttom>
  <buttom>
    <img
      src="avatar.svg"
    />
  </buttom>
</div>
`;
